sdk/servicebus/Azure.Messaging.ServiceBus/tests/ServiceBusTestBase.cs

```diff
@@ -9,15 +9,6 @@ namespace Azure.Messaging.ServiceBus.Tests
 {
     public class ServiceBusTestBase
     {
-        protected static string ConnString = Environment.GetEnvironmentVariable("SERVICE_BUS_CONN_STRING", EnvironmentVariableTarget.Machine);
-        protected static string TenantId = Environment.GetEnvironmentVariable("TENANT_ID", EnvironmentVariableTarget.Machine);
-        protected static string ClientId = Environment.GetEnvironmentVariable("CLIENT_ID", EnvironmentVariableTarget.Machine);
-        protected static string ClientSecret = Environment.GetEnvironmentVariable("CLIENT_SECRET", EnvironmentVariableTarget.Machine);
-        protected const string QueueName = "josh";
-        protected const string SessionQueueName = "joshsession";
-        protected const string TopicName = "joshtopic";
-        protected const string Endpoint = "jolovservicebus.servicebus.windows.net";
-
         protected IEnumerable<ServiceBusMessage> GetMessages(int count, string sessionId = null, string partitionKey = null)
```

Comment:
I believe these are more appropriate via containment, as part of a test utility class rather than using an inheritance chain to share functionality.   Generally, that's considered an anti-pattern in OO.     That said, I'm going to leave them as-is for now to try and avoid orthogonal changes.   I'm keeping a note for a future refactoring pass when time allows.